tes_client/messaging/message_factory.py

```diff
@@ -386,6 +386,31 @@ def cancel_order_capnp(
     return tes_message, cancel_order
 
 
+def cancel_all_orders_capnp(
+        request_header: RequestHeader,
+        account_info: AccountInfo,
+        symbol: str = None,
+        side: str = None):
+    """
+    Generates a capnp CancelAllOrders message.
+    :param request_header: Header parameter object for requests.
+    :param account_info: (AccountInfo) Account on which to cancel order.
+    :param symbol: str (optional)
+    :param side: str (optional)
+    :return: (capnp._DynamicStructBuilder) TradeMessage capnp object,
+             (capnp._DynamicStructBuilder) cancelOrder capnp object.
+    """
+    tes_message, body = _generate_tes_request(request_header=request_header)
+    cancel_all_orders = body.init('CancelAllOrders')
```

Comment:
This will fail in tests - it should be `cancelAllOrders`

Reply:
Good catch.